### DISCUSSION TITLE
XML Format Writer

### DISCUSSION THREAD

**Discussion by djlift** [PHPOffice/PhpSpreadsheet#4229] [Q&A] [answered]
Hello,
I have a need to write XML format files and PhpSpreadsheet only writes xlsx files right now.

Supposedly the xlsx version that PhpSpreadsheet writes is just a zipped version of openxml format, but if i unzip the xlsx file, it is not the same as saving an xlsx doc to xml format using the Excel. Is there any way to accomplish this?
Thanks :)

**Comment by oleibman** [comment, marked as answer]
PhpSpreadsheet writes several formats besides Xlsx. Unfortunately, Xml files are supported only for read.
